tdrs-frontend/src/components/EditProfile/EditProfile.test.js

```diff
@@ -19,19 +19,6 @@ describe('EditProfile', () => {
   }
   const mockStore = configureStore([thunk])
 
-  it('should have a card with header Request Access', () => {
-    const store = mockStore(initialState)
-    const wrapper = mount(
```

Comment:
This test is no longer relavent to only edit profile page. We run this test with private template